sympy/printing/pretty/pretty.py

```diff
@@ -882,19 +882,18 @@ def _print_BlockMatrix(self, B):
         return self._print(B.blocks)
 
     def _print_MatAdd(self, expr):
-        s = None
-        for item in expr.args:
-            pform = self._print(item)
-            if s is None:
-                s = pform     # First element
-            else:
-                coeff = item.as_coeff_mmul()[0]
-                if S(coeff).could_extract_minus_sign():
-                    s = prettyForm(*stringPict.next(s, ' '))
-                    pform = self._print(item)
-                else:
-                    s = prettyForm(*stringPict.next(s, ' + '))
-                s = prettyForm(*stringPict.next(s, pform))
+        args = list(expr.args)
```

Comment:
why is this done?

Reply:
Only because I saw it lower down, and I didn't know for sure that it would be an indexable object.  If it will, then can be omitted, of course.

Reply:
expr.args will always be an iterable, yes.

Reply:
Will it always be indexable?  So I can do `expr.args[0]` and `expr.args[1:]`?

Reply:
args is always a list (or possibly a tuple, I do not recall). 

It may be of length 1 though, e.g. `MatAdd(A)` will not evaluate to `A` (which is probably a deficiency, but then there is still the possibility that someone does `MatAdd(A, evaluate=False)`). Seems like `MatAdd(evaluate=False)` does evaluate to `GenericZeroMatrix`, so one can be sure that it is at least of length 1.

Reply:
Ah - length 1 is OK:

```python
[ins] In [1]: a = [1]

[ins] In [2]: a[1:]
Out[2]: []
```